packages/www/pages/dashboard/streams/[id]/health.tsx

```diff
@@ -0,0 +1,201 @@
+import React, { useCallback } from "react";
+import { Box, Flex, Link, Text } from "@livepeer.com/design-system";
+import Layout from "../../../../layouts/dashboard";
+import { useRouter } from "next/router";
+import { useApi } from "../../../../hooks";
+import { useEffect, useState } from "react";
+
+import Chart from "@components/Chart";
```

Comment:
Can we recreate this same component inside `Dashboard` directory and refactor it to use stitches? (Replace `div` with `Box`, `sx=` with `css=`, use the correct violet color, etc.